datacatalog/template.yml

```diff
@@ -4,7 +4,8 @@ Description: AWS SAM template for the AWS::Athena::DataCatalog resource type
 
 Globals:
   Function:
-    Timeout: 60  # docker start-up times can be long for SAM CLI
+    Timeout: 180  # docker start-up times can be long for SAM CLI
```

Comment:
why we change this one to 180? Any context?

Reply:
For the SAM template to run contract tests, in my experience there are timeout issues with the default 60 seconds so I increased it to 180 here.